### PR TITLE
refactor: replace params method in PromuSlice

### PR DESCRIPTION
**Pull request by baudoliver7**
A new method has been introduced in `RqParams` (artipie/http) that allow us to obtain all values of a
query parameter. We refactored `PromuSlice` with this new method.

Ref: #978

### CODE REVIEW

**Comment by baudoliver7** [comment]
@artipie/maintainers ready to be reviewed :)

**Review by genryxy — APPROVED** [review]
@baudoliver7 thanks, looks good to me
@g4s8 please check

**Comment by baudoliver7** [comment]
@g4s8 waiting to be merged

File: src/main/java/com/artipie/http/PromuSlice.java [hunk 1]
```diff
@@ -54,8 +51,12 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
...
         final String ctype = new Accept(headers).values().get(0);
         try (StringWriter writer = new StringWriter()) {
-            final Set<String> filters = PromuSlice.params(line, "name");
-            final MetricsOutput output = new PrometheusOutput(writer, ctype, filters);
+            final List<? extends String> filters = new RqParams(
+                new RequestLineFrom(line).uri()
+            ).values("name");
+            final MetricsOutput output = new PrometheusOutput(
+                writer, ctype, new HashSet<>(filters)
+            );
```
Comment (g4s8) [inline src/main/java/com/artipie/http/PromuSlice.java:59]: It seems we use `filters` only once, I'd suggest inline this variable

**Review by g4s8 — CHANGES_REQUESTED** [review]
@baudoliver7 thanks, please check my comment

File: src/main/java/com/artipie/http/PromuSlice.java (hunk 1 above)
Comment (baudoliver7) [reply src/main/java/com/artipie/http/PromuSlice.java]: fixed

**Comment by baudoliver7** [comment]
@g4s8 I took advantage to bump `pdd-action` from `v1.0.0` to `master`. All things look good now :)

**Review by g4s8 — APPROVED** [review]
@baudoliver7 thanks

**Pull request merged**